docs/quickstarts/insights-malware-install/insights-malware-install.yml

```diff
@@ -0,0 +1,206 @@
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+
+    - title: Install YARA
+      description: |-
+
+        YARA is an open-source and multi-platform tool that detects code similarities between predefined malware signatures and a  scanned RHEL system. 
+        If one or more conditions in a predefined signature exist on a scanned RHEL system. YARA records a malware hit on that system. 
+        
+        **NOTE**: Install the YARA Package on each host you want to scan for Malware and Ensure the System’s Operating System versions are either RHEL 8 or RHEL 9.  
+
+        1. Install YARA: 
+        ```
+        sudo dnf install yara 
+        ```   
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn't verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+ 
+    - title: Install Insights Client and Register to RHEL 
+      description: |- 
+         1. Install Insights client:
+         ```
+         sudo dnf install insights-client 
+         ```   
+         2. Test the Insights client connection:
+         ```
+         sudo insights-client --test-connection
+         ```   
+         3. Register the system with Insights for Red Hat Enterprise Linux :
+         ```
+         sudo insights-client --register
+         ```      
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn't verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+
+    -  title: Run the Malware detection collector 
+       description: |-
+        Run the Malware detection tool and connect to Insights.
+       introduction: |-
+        **Welcome to the quick start for running the malware detection tool and connecting to Insights. Make sure to complete **The Configure User Access quick start** before attempting this quick start.
+        In this quick start, you will run the client malware detection collector on your hosts, make the scans live on Insights, and complete full filesystem scans. 
+     
+    - title: Initial run of the Malware detection collector 
+      description: |-
+        1. Run the Insights client malware detection collector:
+        ```
+        sudo insights-client --collector malware-detection
+        ```   
+        The initial run creates a malware detection configuration file in 
+        ```
+        /etc/insights-client/malware-detection-config.yml
```

Comment:
I believe if you use the backticks around the filename, you could mark this up as code, without it showing as a new line (so you can delete the 'new line' markup) and it would be one sentence. Something like this:
```suggestion
        `/etc/insights-client/malware-detection-config.yml`
```